### PR TITLE
Bump to v2.0.2

### PR DESCRIPTION
**Pull request by bodono**
Now working on windows again. Update the tags to the latest release.

### CODE REVIEW

**Comment by mlubin** [comment]
So what type is used for integers now? If it's `long`, then we should switch Julia's `Int` to `Clong` in the appropriate places for cross-platform compatibility.

File: deps/build.jl [hunk 1]
```diff
@@ -19,7 +19,7 @@ if is_apple()
 end
 
 version = "2.0.2"
-win_version = "2.0.2" # 2.0.* is not yet working on windows
+win_version = "2.0.2"
```
Comment (mlubin) [inline deps/build.jl]: Why remove this comment when the current build script still does not support windows?

**Comment by mlubin** [comment]
I see that we switched from `Clong` to `Int` some time ago... https://github.com/JuliaOpt/SCS.jl/commit/51cc9f0061cee0d0a682c191af7ab06e1d977b9c. Not sure why.

File: deps/build.jl (hunk 1 above)
Comment (bodono) [reply deps/build.jl]: SCS 2.0.2 works fine with windows, the binaries need to be updated for SCS.jl. I updated the comment.

Comment (mlubin) [reply deps/build.jl]: Ok, good to clarify.

**Pull request merged**